app/views/feeds/_form.html.erb

```diff
@@ -1,4 +1,8 @@
-<%= form_with(url: podcast_feed_path(podcast, feed), model: feed, html: {autocomplete: "off"}, data: {controller: "unsaved disable", unsaved_changed_value: feed.changed?}, unsaved_confirm_value: t(".confirm")) do |form| %>
+<% model = feed.new_record? ? [feed.podcast, feed] : feed %>
+<% url = feed.new_record? ? podcast_feeds_path(feed.podcast) : podcast_feed_path(feed.podcast, feed) %>
+<% method = feed.new_record? ? :post : :put %>
+
+<%= form_with(url: url, model: model, method: method, html: {autocomplete: "off"}, data: {controller: "unsaved", unsaved_changed_value: feed.changed?}, unsaved_confirm_value: t(".confirm")) do |form| %>
```

Comment:
You're missing the `disable` controller on this form?  The image uploads are complaining about it.  I might also pull all of this data up into a `<% data = {...} %>` above, to shorten the wrapping.

(I think I added the `disable` controller in a recent PR, to disable the save button while uploads are still in-progress).

Reply:
Hmm... there's also something odd going on with the "unsaved" controller here?  If I make a change, the field gets marked with the orange outline, but it lets me navigate away without prompting me "you have unsaved changes".

Maybe adding the "disable" controller will help?